src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.ShoppingCartDao;
+import mate.academy.dao.TicketDao;
+import mate.academy.lib.Inject;
+import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.ShoppingCartService;
+
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private TicketDao ticketDao;
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        Ticket ticketFromDb = ticketDao.add(ticket);
+        ShoppingCart shoppingCartOfUser = shoppingCartDao.getByUser(user).get();
+        List<Ticket> ticketsOfUser = shoppingCartOfUser.getTickets();
+        ticketsOfUser.add(ticketFromDb);
```

Comment:
насправді нам не обовязково створювати окрему змінну
```suggestion
        ticketDao.add(ticket);
        ShoppingCart shoppingCartOfUser = shoppingCartDao.getByUser(user).get();
        List<Ticket> ticketsOfUser = shoppingCartOfUser.getTickets();
        ticketsOfUser.add(ticket);
```